src/Flexdis86/InstructionSet.hs

```diff
@@ -294,6 +295,11 @@ ppInstruction i =
             ([], _) -> sLockPrefix <+> text op
             (_,_)   -> sLockPrefix <+> text op <+> ppPunctuate comma (ppValue <$> args)
 
+-- | Allows libraries using `ppInstruction` to not have to depend on deprecated
+-- ansi-wl-pprint to turn instructions into `String`s.
```

Comment:
Echoing @kquick's comments, it would be nice to remove this comment about `ansi-wl-pprint` once we remove the `ansi-wl-pprint` dependency.